ckan/templates/package/snippets/additional_info.html

```diff
@@ -60,13 +60,22 @@
         {% if pkg_dict.metadata_modified %}
           <tr>
             <th scope="row" class="dataset-label">{{ _("Last Updated") }}</th>
-            <td class="dataset-details">{{ h.render_datetime(pkg_dict.metadata_modified, with_hours=True) }}</td>
+            <td class="dataset-details">
+                <span class="datetime" data-datetime="{{ h.render_datetime(pkg_dict.metadata_modified, date_format='%Y-%m-%dT%H:%M:%S%z') }}">
```

Comment:
This should always work, but I yes, it seems a _bit_ complicated. If I should change that, I guess I would have to introduce a new argument for `render_datetime` called `use_utc=True` or similar.